index.html

```diff
@@ -4,24 +4,56 @@
 <head>
 <meta charset="utf-8">
 <link rel="stylesheet" href="index.css">
-
-<!-- development version, includes helpful console warnings -->
-<script src="https://cdn.jsdelivr.net/npm/vue@2/dist/vue.js"></script>
-
 <body>
+
 <main id="chargen">
+
     <h1>CharGen</h1>
-    <div>
+
+    <div id="Info" class="clear">
```

Comment:
I was thinking if we need id here. Like id="Info" - I suppose it seems to be useful to have it for future and also it's logical to have it, but do we actually use it?

Same for other cases bellow, Attributes, Skills and other